examples/with-npm/apps/web/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "tsconfig/nextjs.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "compilerOptions": {
+    "plugins": [{ "name": "next" }]
+  },
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
```

Comment:
Can we move these to the shared next ts config?

Reply:
No, unfortunately!
- The plugin has to be directly in the Next.js project to get detected.
- The `includes` that `next` cares about get reinserted when you kick off a build.